src/compute-client/src/explain.rs

```diff
@@ -255,13 +253,10 @@ pub struct TimestampSource<T> {
 
 impl<T: fmt::Display + fmt::Debug> fmt::Display for TimestampExplanation<T> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        writeln!(f, "     timestamp: {:13}", self.timestamp)?;
-        writeln!(f, "         since:{:13?}", self.since)?;
-        writeln!(f, "         upper:{:13?}", self.upper)?;
-        writeln!(f, "     has table: {}", self.has_table)?;
-        if let Some(ts) = &self.table_read_ts {
-            writeln!(f, " table read ts: {:13}", ts)?;
-        }
+        writeln!(f, "                timestamp: {:13}", self.timestamp)?;
+        writeln!(f, "                    since:{:13?}", self.since)?;
+        writeln!(f, "                    upper:{:13?}", self.upper)?;
+        writeln!(f, "  can respond immediately: {}", self.respond_immediately)?;
```

Comment:
I also kinda want, in the case when it can't respond immediately, the diff between `timestamp` and `upper` so it's quick to eyeball how far behind it is.

Reply:
Ah yeah, maybe "Inherent delay: { 1234ms, None }" or something similar?

Reply:
This has been taking me a surprising long amount of time. I'm going to open an issue for further improvements and include this.

Reply:
https://github.com/MaterializeInc/materialize/issues/14947